xknx/devices/binary_sensor.py

```diff
@@ -80,6 +80,11 @@ def unique_id(self) -> str | None:
         """Return unique id for this device."""
         return f"{self.remote_value.group_address_state}"
 
+    @property
+    def last_telegram(self) -> Optional["Telegram"]:
```

Comment:
```suggestion
    def last_telegram(self) -> Telegram | None:
```
or `"Telegram" | None` depending on if it is imported directly or only at TYPE_CHECKING. (I'd just import it if it isn't).